lib/improver/tests/threshold/test_BasicThreshold.py

```diff
@@ -648,15 +654,15 @@ def test_invalid_lower_bound(self):
             Threshold(threshold,
                       fuzzy_bounds=fuzzy_bounds)
 
-    def test_invalid_threshold_method(self):
-        """Test plugin throws a ValueError when threshold_method is bad"""
-        threshold_method = 'invalid'
+    def test_invalid_inequality(self):
+        """Test plugin throws a ValueError when inequality is bad"""
+        inequality = 'invalid'
         threshold = 0.6
-        msg = ('String "{}" does not match any known threshold method'.format(
-            threshold_method))
+        msg = ('String "{}" does not match any known inequality method'.format(
+            inequality))
         with self.assertRaisesRegex(ValueError, msg):
             Threshold(threshold,
-                      threshold_method=threshold_method)
+                      inequality=inequality)
```

Comment:
Perhaps the indentation could be amended here? I.e.
```python
Threshold(threshold, inequality=inequality)
```
Also for the few unit tests above i.e.
```python
Threshold(threshold, fuzzy_bounds=fuzzy_bounds)
```

Reply:
Compressed to one line in several places as suggested.